cmake/AddEpubTarget.cmake

```diff
@@ -48,11 +47,35 @@ function (add_epub_target docname lang entities figures dtd_files)
         COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/OEBPS/images/callouts"
         COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger")
 
+    # Copy figures for this document
+    add_custom_command(
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-fig-trigger"
+        COMMAND ${CMAKE_COMMAND} -E copy ${figures} "${BUILD_DIR}/OEBPS/figures"
+        COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-fig-trigger"
+        DEPENDS ${figures}
+                "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger")
+
+    # Copy XSL Stylesheet icons
+    add_custom_command(
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-xslticon-trigger"
+        # PNG admonition icons are used in EPUB.
+        COMMAND cp -f "${CMAKE_SOURCE_DIR}/xsl/images/*.png" "${BUILD_DIR}/OEBPS/images"
+        # SVG callout icons are used in EPUB.
+        COMMAND cp -f "${CMAKE_SOURCE_DIR}/xsl/images/callouts/*.svg" "${BUILD_DIR}/OEBPS/images/callouts"
+        COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-xslticon-trigger"
+        DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger")
+
+    # Copy GnuCash-Specific icons
+    add_custom_command(
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-gnucashicon-trigger"
+        COMMAND cp -f "${CMAKE_SOURCE_DIR}/xsl/icons/*" "${BUILD_DIR}/OEBPS/images"
+        COMMAND touch "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-gnucashicon-trigger"
+        DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-xslticon-trigger"
+                "${gnucash_icon_files}")
+
 
     add_custom_command(
         OUTPUT "${BUILD_DIR}/${epubfile}"
-        COMMAND rm -fr "${EPUB_TMPDIR}"
-        COMMAND mkdir "${EPUB_TMPDIR}"
```

Comment:
(Nitpick) Not removing the build directory means your final epub document may contain html files and or images that are not used because we zip up the content of that directory unconditionally. This can happen if:
* the xsl transform changes the chunking (eg sections get renamed)
* images are removed from the source

That's the reason also the command depends on the figure source files rather than the figtrigger.

It's a minor issue, which I think I'll let pass. Builders can avoid this by building in a clean build directory.